pkg/kbld/cmd/unprocessed_image_urls.go

```diff
@@ -33,3 +35,7 @@ func (i *UnprocessedImageURLs) All() []UnprocessedImageURL {
 	})
 	return result
 }
+
+func (i *UnprocessedImageURLs) Bytes() ([]byte, error) {
```

Comment:
dont think we want to use `url: ...` in its output. prolly better to go with `image: ...`

Reply:
I think, for now, placing the json struct tags on the internal data structure is _sufficient_ for now. Might be something to revisit as other requests (like #95) are addressed.